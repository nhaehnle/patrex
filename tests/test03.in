{
	obj->setup(a.foo, b, c, d, e, f.bar(), g.test(1, 2), h);
}
