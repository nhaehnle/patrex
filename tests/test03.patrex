
match ${id}|varname| ->setup( $.* $<|prevarg| , $!(,)*|arg|, $!(,)*, $!(,)* ); $<|pos|
	erase prevarg arg.end
	insert pos "\n\t{varname}->set({arg});"
