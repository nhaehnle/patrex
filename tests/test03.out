{
	obj->setup(a.foo, b, c, d, e, g.test(1, 2), h);
	obj->set(f.bar());
}
